mne/utils/check.py

```diff
@@ -65,14 +65,14 @@ def check_fname(fname, filetype, endings, endings_err=()):
     if len(endings_err) > 0 and not fname.endswith(endings_err):
         print_endings = " or ".join([", ".join(endings_err[:-1]), endings_err[-1]])
         raise OSError(
-            "The filename (%s) for file type %s must end with %s"
-            % (fname, filetype, print_endings)
+            f"The filename ({fname}) for file type {filetype}"
```

Comment:
Looking at this line for example
https://github.com/mne-tools/mne-python/actions/runs/8055841114/job/22004174246?pr=12464#step:17:9757

You see
```
FAILED mne/io/tests/test_raw.py::test_test_raw_reader - AssertionError: Regex pattern did not match.
 Regex: 'raw must end with .fif or .fif.gz'
 Input: 'The filename (/tmp/tmp_mne_tempdir_euhw91nk/test_raw.h5) for file type rawmust end with .fif or .fif.gz'
```
You can see the problem -- there is a missing space. So you can fix with:
```suggestion
            f"The filename ({fname}) for file type {filetype} "
```